### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by Shujun-He** [csrhddlam/axial-deeplab#20]
Can you release more pretrained weights in addition to Axial-ResNet-26-S?

**Comment by csrhddlam** [comment]
Sorry we don't have free GPU resources to run more models, but please let me know if you have run some other models.

**Comment by Shujun-He** [comment]
I have a 2x3090 machine so I can run some models. Do I just run the train script?

**Comment by csrhddlam** [comment]
Sounds great!

Yes, just follow the command line in the README, but you might need to linearly scale the learning rate and batch size accordingly.
